mlflow/utils/search_utils.py

```diff
@@ -1500,6 +1500,13 @@ class SearchTraceUtils(SearchUtils):
         "execution_time",
     }
 
+    # Databricks backend does not support LIKE/ILIKE operators for trace search
+    # for performance issues, so we don't support them in the OSS either for
+    # consistency. We can revisit this decision if we find a way to support
```

Comment:
```suggestion
    # For now, don't support LIKE/ILIKE operators for trace search because it may
    # cause performance issues with large attributes and tags. We can revisit this
    # decision if we find a way to support them efficiently.
```

Reply:
(Databricks isn't the primary reason)